wf-core-tck-runner/README-WFP.adoc

```diff
@@ -0,0 +1,70 @@
+= WildFly Preview Notes
```

Comment:
LATER: We should update this page to focus on standard WildFly.

Reply:
Addressed